content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md

```diff
@@ -0,0 +1,142 @@
+
+- **Fixed** a bug that would ignore some configuration when specifying multiple `accessLogging` in Telemetry resources within a single stanza. With this fix, all provided access logging configuration within a single stanza of Telemetry resource are respected.
+  ([Issue #39468](https://github.com/istio/istio/issues/39468))
+
+## Extensibility
+
+- **Added** the `WASM_MODULE_EXPIRY`, `WASM_PURGE_INTERVAL`, `WASM_HTTP_REQUEST_TIMEOUT`, and `WASM_HTTP_REQUEST_MAX_RETRIES` istio-agent environment variables to control WASM cache related parameters.
+
+- **Added** the ability to decompress and/or untar the WASM binary when it is pulled via HTTP/HTTPS.
+
+- **Added** the `WASM_INSECURE_REGISTRIES` istio-agent environment variable for when the `WasmPlugin` is pointing HTTP/HTTPS server.
+
+- **Extended** the scope of `ImagePullPolicy` in `WasmPlugin` to accept HTTP/HTTPS URLs in addition to OCI image URLs.
+
+## Installation
+
+- **Added** support for `arm64` architecture for all components.
+  ([Issue #26652](https://github.com/istio/istio/issues/26652))
+
+- **Added** `--log_output_level` and `--log_as_json` to the `istio-init` container (as they are in `istio-proxy`).
+
+- **Updated** the default value of the feature flag `ENABLE_LEGACY_FSGROUP_INJECTION` to false.
+This may cause issues with sidecars when installing on Helm on Kubernetes versions prior to 1.19.
+
+- **Updated** the Kiali addon to the latest version (v1.55).
+
+- **Added** support for watching local secret resource updates for external istiod.
+  ([Issue #31946](https://github.com/istio/istio/issues/31946))
+
+- **Added** values to the Istio Gateway Helm chart for configuring [topologySpreadConstraints](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/) on the gateway deployment.
+
+- **Removed** the deprecated `remote.yaml` profile which is equivalent to the default profile.
+  ([Issue #38832](https://github.com/istio/istio/issues/38832))
+
+## istioctl
+
+- **Promoted** `istioctl x uninstall` to `istioctl uninstall`.
+  ([Issue #40339](https://github.com/istio/istio/issues/40339))
+
+- **Improved** the output format of the active logging levels.
+
+- **Added** a new analyzer for Envoy filter patch operations to provide warnings when relative patch operations are used without a priority set which can cause Envoy filters not to be applied correctly.
+  ([Issue #37415](https://github.com/istio/istio/issues/37415))
+
+- **Added** `istioctl analyze` beta API version support for file resources.
+
+- **Added** pod name and cluster name to bookinfo's reviews, where the cluster name is determined by the `CLUSTER_NAME` environment variable on the reviews deployments.
+
+- **Added** support for parsing list type of files in `istioctl analyze`.
+  ([Issue #39982](https://github.com/istio/istio/issues/39982))
+
+- **Added** description to `istioctl admin log`
```

Comment:
Add a period to the send of the sentence.

Reply:
Done.